corehq/apps/tour/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    web_user = models.CharField(max_length=80, null=False)
+    tours = json_field.JSONField(
+        default={},
+    )
+    history = json_field.JSONField(
+        default={},
+    )
+
+    @classmethod
+    def enable_tour_for_user(cls, tour_slug, web_user):
+        # todo
+        pass
+
+    @classmethod
+    def is_tour_enabled_for_user(cls, tour_slug, web_user):
+        # todo
+        return True
```

Comment:
would be useful to see how these two functions get implemented.

if you make the above `seen_tours` (and maybe make it a `set`?) then you can just do `tour_slug in cls.objects.get(user=user).seen_tours` (with error handling and what not). if it's `needed_tours` then it's kind of the inverse. 

the nice thing about using `seen_tours` would be that you wouldn't have to resave every `GuidedTours` object each time we added a tour.